lua/ssr.lua

```diff
@@ -56,13 +55,18 @@ function Ui.new()
 
   self.origin_win = api.nvim_get_current_win()
   local origin_buf = api.nvim_win_get_buf(self.origin_win)
-  self.lang = parsers.get_buf_lang(origin_buf)
-  if not parsers.has_parser(self.lang) then
+  local lang = vim.treesitter.language.get_lang(vim.api.nvim_get_option_value("ft", { buf = buf }))
+  if not lang then
```

Comment:
Seems `ts.language.get_lang()` won't return nil when the parser is not installed, we probably need to call `vim._ts_has_language()` since nvim-treesitter's `parsers.has_parser()` used that.

Reply:
I wouldn't recommend using internals, usage of that is removed in the rewrite